packages/api-bindings/src/graphql/__helpers__/mutations.ts

```diff
@@ -418,3 +421,29 @@ export function createCreateSetProfileMetadataViaDispatcherMutationMockedRespons
     result: { data },
   };
 }
+
+export function createReportPublicationMutationMockedResponse(args: {
+  variables: ReportPublicationMutationVariables;
+}): MockedResponse<ReportPublicationMutation> {
+  return {
+    request: {
+      query: ReportPublicationDocument,
+      variables: args.variables,
+    },
+    result: { data: { reportPublication: null } },
+  };
+}
+
+export function createReportPublicationMutationWithErrorMockedResponse(args: {
+  variables: ReportPublicationMutationVariables;
+}): MockedResponse<ReportPublicationMutation> {
+  return {
+    request: {
+      query: ReportPublicationDocument,
+      variables: args.variables,
+    },
+    result: {
+      errors: [new GraphQLError('Publication already reported')],
```

Comment:
I think the mocked error response is correct..

Reply:
If I log the output of `console.log(e instanceof ApolloError, e);` in the `catch` part of `ReportPublicationGateway.ts` I'm getting:

```
false ApolloError: Publication already reported
        at new ApolloError (...)
   (...)
   {
      graphQLErrors: [ GraphQLError [Object] { message: 'Publication already reported' } ],
      clientErrors: [],
      networkError: null,
      extraInfo: undefined
    }
```
so it looks fine but the `instanceof` check is returning `false`

I don't know why

Reply:
@krzysu PR that's fixing the issue https://github.com/lens-protocol/lens-sdk/pull/103

Reply:
I checked, and indeed that solves the issue, thx!